packages/grafana-toolkit/src/cli/tasks/plugin.build.ts

```diff
@@ -149,7 +149,9 @@ export const lintPlugin = useSpinner<Fixable>('Linting', async ({ fix }) => {
 
   if (lintResults.length > 0) {
     console.log('\n');
-    const failures: RuleFailure[] = lintResults.flat();
```

Comment:
What was the issue with `Array::flat`? We use `typescript@3.7.x` which supports ES2019 features.

Reply:
Nothing apart from flattening it in a wrong way. It's an array of objects that has the arrays nested. And those need to be flattened. But, since ESLint is just around the corner I don't want to focus more on this rather than simple revert to make it working in 6.6.1. Hope this is fine.